RecoTracker/LSTCore/standalone/code/core/write_lst_ntuple.cc

```diff
@@ -944,30 +944,30 @@ std::tuple<float, float, float, std::vector<unsigned int>, std::vector<unsigned
 
 //________________________________________________________________________________________________________________________________
 void printHitMultiplicities(Event* event) {
-  Modules const* modules = event->getModules().data();
-  ObjectRanges const* ranges = event->getRanges().data();
+  auto modules = event->getModules<ModulesSoA>();
+  auto ranges = event->getRanges<ObjectRangesSoA>();
 
   int nHits = 0;
-  for (unsigned int idx = 0; idx <= *(modules->nLowerModules);
+  for (unsigned int idx = 0; idx <= modules.nLowerModules();
        idx++)  // "<=" because cheating to include pixel track candidate lower module
   {
-    nHits += ranges->hitRanges[4 * idx + 1] - ranges->hitRanges[4 * idx] + 1;
-    nHits += ranges->hitRanges[4 * idx + 3] - ranges->hitRanges[4 * idx + 2] + 1;
+    nHits += ranges.hitRanges()[2 * idx][1] - ranges.hitRanges()[2 * idx][0] + 1;
+    nHits += ranges.hitRanges()[2 * idx + 1][1] - ranges.hitRanges()[2 * idx + 1][0] + 1;
```

Comment:
from the comment in the ranges.h, this seems like a dead code

Reply:
I ended up just fixing it so that it uses the right array. There's a bunch more `print*` functions that are dead code, but they could be potentially useful for debugging.